contents/blog/posthog-vs-mixpanel.md

```diff
@@ -0,0 +1,925 @@
+---
+date: 2022-12-05
+title: PostHog vs Mixpanel
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Guides", "Comparisons"]
+---
+
+Want to know how PostHog and Mixpanel are different? If you remember nothing else, remember these two points:
+
+1. Mixpanel is a product analytics tool.
+
+2. PostHog is an all-in-one platform that _includes_ product analytics **and** a whole bunch more.
+
+In this guide, we'll cover:
+
+- Mixpanel and PostHog's [core features](#core-features)
+- [Privacy and compliance](#privacy--compliance) on each platform
+- How they track usage and [library support](#tracking--sdks)
+- [Compare available integrations](#integrations) in multiple categories
+- [Answer frequently asked questions](#frequently-asked-questions) about PostHog and Mixpanel
+
+## How is PostHog different to Mixpanel?
+
+### 1. PostHog does more than product analytics
+
+Don't mistake us, product analytics is a core part of our platform, but it's just one cog in a suite of vital tools for product and data teams. When you adopt PostHog you also get session recording (with console logs), feature flagging, A/B and multivariate testing, and customizable data pipelines – all seamlessly integrated and with just one bill to pay. Don't want session recording? Just turn it off.
+
+### 2. It's built for engineers
+
+Anyone can use PostHog – just [ask our marketing team](/blog/posthog-vs-amplitude) – but it's designed for engineers first. We believe engineering-led companies are the future of tech and that future requires tools that let software engineers flex their product muscles. In building a platform [product engineers](/blog/what-is-a-product-engineer) love, we're also building a platform that gives data and marketing teams unrivaled insights.
+
+### 3. We're an open book
+
+PostHog is [built on transparency](/blog/how-to-run-a-transparent-company). Our product is open source and released under an MIT License. You can read [our source code](https://github.com/PostHog/posthog), raise your own issues and PRs, and [build apps atop PostHog](/blog/how-we-built-an-app-server) to support custom data flows. Anyone can read our [company handbook](/handbook) and [how we pay people](/handbook/people/compensation).
+
+> This guide compares Mixpanel's self-serve Growth plan with PostHog Cloud. Features available on other plans are marked as such.
+
+## Features
```

Comment:
I think it could even be worth starting with a high level comparison table. Something like the below, but obviously expanded a bit and styled better. 

|                         | **PostHog** | **Mixpanel** |
|-------------------------|-------------|--------------|
| **Product analytics**   |     YES     |      YES     |
| **Session recording**   |     YES     |      NO      |
| **Feature flags**       |     YES     |      NO      |
| **A/B experimentation** |     YES     |      NO      |

Reply:
Might give this a go. This also permits going a bit deeper on these further down.